src/main/java/blackjack/domain/game/Referee.java

```diff
@@ -0,0 +1,53 @@
+package blackjack.domain.game;
+
+import static java.util.stream.Collectors.counting;
+import static java.util.stream.Collectors.groupingBy;
+
+import blackjack.domain.participant.Participant;
+import blackjack.domain.participant.Players;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Referee {
+
+    private static final int BURST_SCORE = 21;
```

Comment:
```suggestion
    private static final int BURST_SCORE = 21;
```

21은 blackjack 점수이지, burst 점수는 아닌거 같아요. 22아닐까요?